app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+import shutil
+
+
+def copy_file(command: str):
+    new_command = command.split()
+    file = new_command[1]
+    new_file = new_command[2]
+
+    if file != new_file:
+        shutil.copy(file, new_file)
```

Comment:
In practice, you can use this module
But the task was to copy a file using file.read and file.write methods,